SonarQube.Client/Api/V7_20/GetIssuesRequest.cs

```diff
@@ -80,11 +80,11 @@ protected override SonarQubeIssue[] ParseResponse(string response)
         }
 
         private SonarQubeIssue ToSonarQubeIssue(ServerIssue issue) =>
-            new SonarQubeIssue(FilePathNormalizer.NormalizeSonarQubePath(ComputePath(issue.Component)), issue.Hash, issue.Line, issue.Message, ComputeModuleKey(issue),
+            new SonarQubeIssue(ComputePath(issue.Component), issue.Hash, issue.Line, issue.Message, ComputeModuleKey(issue),
                 GetRuleKey(issue.CompositeRuleKey), issue.Status == "RESOLVED", ToIssueFlows(issue.Flows));
 
         private string ComputePath(string component) =>
-            componentKeyPathLookup[component].FirstOrDefault() ?? string.Empty;
+            FilePathNormalizer.NormalizeSonarQubePath(componentKeyPathLookup[component].FirstOrDefault() ?? string.Empty);
```

Comment:
I missed this when I added FilePathNormalizer but flows were never used up until now so no harm was done.